lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -113,6 +113,16 @@ static public function addGlobalIgnoredName($name)
     }
 
     /**
+     * Gets the globally ignored annotation names.
+     *
+     * @return array An array of Annotation names to ignore.
+     */
+    static public function getGlobalIgnoredNames()
```

Comment:
The need for this smells of encapsulation breakage: there should be a solution that doesn't involve accessing the `AnnotationReader` directly. Maybe mirror the same structure inside the `SimpleAnnotationReader`?